src/core/middleware.py

```diff
@@ -51,8 +51,8 @@ def __init__(self, get_response):
     def __call__(self, request):
         public_headers = {
             'public': True,
-            'max-age': 60 * 5, # 5 minutes, 300 seconds
-            'stale-while-revalidate': 60 * 5, # 5 minutes, 300 seconds
+            'max-age': settings.CACHE_HEADERS_TTL,
+            'stale-while-revalidate': settings.CACHE_HEADERS_TTL,
```

Comment:
We might want to set this to 0 in `end2end`, should force the refresh before looking at the page rather than after.

Reply:
Is it totally equivalent to a page that gets cached and then expire very soon? End2end knows how to deal with cached pages by continuing to poll for a new result.

Reply:
Yeah, might just be slightly quicker (avoid some polling, but the request from Journal happens when producing the page - depends how often it polls).